bench-scripts/hp-brcu/bench-long-running.py

```diff
@@ -1,37 +1,42 @@
 import subprocess
```

Comment:
why we have different bench scripts for different papers? why not having a single set of scripts for both papers?

Reply:
It would be complicated for both clients and developers to provide a single benchmark script because the types and configurations of workloads are different across the papers.

For example, the HP-BRCU paper performs long-running operation experiments that the CIRC paper doesn't, and the CIRC performs queue experiments that other SMR papers do not. Additionally, other configurations (e.g., marker design, time interval to execute, plot sizes) are different because they are specialized for each paper.